theano/configdefaults.py

```diff
@@ -55,6 +55,13 @@ def floatX_convert(s):
              in_c_key=False,
              )
 
+AddConfigVar('pickle_test_value',
+             "Dump test values while pickling model. "
+             "If True, test values will be dumped with model.",
+             BoolParam(False),
```

Comment:
After some discussion with @nouiz, we now believe that `True` would be a better default for the release, in order to avoid breaking anything.

We plan to change it to `False` after the release.